contracts/test/EventTest.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity ^0.5.0;
+
+contract EventTest {
+
+  event Deposit(uint batchNumber ,uint idBalanceTree, uint depositAmount, uint tokenId, uint Ax, uint Ay,
+    address withdrawAddress );
+
+  event ForgeBatch(uint batchNumber, bytes offChainTx);
```

Comment:
Its probably going to be a little more efficient not to put `offChainTx` because we need to gas per per byte for logs. 

Instead we can throw an event with teh block number and look up that block number in the node and parse the data there.

Reply:
Agree!
I will add an issue and notice @jbaylina about your proposal

Reply:
:+1: